pkg/eventprocessor/impl/webhook.go

```diff
@@ -76,14 +80,162 @@ func sendWebhookRequest(ctx context.Context, url string, body interface{}) error
 	return nil
 }
 
+type chain struct {
+	ID               int64
+	Name             string
+	ContractAddr     common.Address
+	TBLDocsURL       string
+	BlockExplorerURL string
+	SupportsNFTView  bool
+}
+
+var chains = map[int64]chain{
```

Comment:
What happens if you change to `map[tableland.ChainID]chain`? does it avoid the `int64` conversions?